pkg/ride/generate/internal/functions_generation.go

```diff
@@ -919,12 +931,24 @@ func evaluationCatalogueV7EvaluatorV1() map[string]int {
 	return m
 }
 
+func evaluationCatalogueV8EvaluatorV1() map[string]int {
```

Comment:
Please, remind me, why we need `evaluationCatalogueEvaluatorV1` for all versions of ride since RideV6 included?

I can see the code inside `newEvaluationScope`:
```go
	ev := 1
	if env.rideV6Activated() {
		ev = 2
	}
	costs, err := selectEvaluationCostsProvider(v, ev)
	if err != nil {
		return evaluationScope{}, err
	}
```
Inside `selectEvaluationCostsProvider`:
```go
	switch v {
	case ast.LibV1, ast.LibV2:
		switch ev {
		case 1:
			return EvaluationCatalogueV2EvaluatorV1, nil
		default:
			return EvaluationCatalogueV2EvaluatorV2, nil
		}
		case ...
```
It means that we need catalogueV1 for evaluator V1. But `RideV6` must be activated for scripts V6 and above.
And `EvaluationCatalogueV6EvaluatorV1`, `EvaluationCatalogueV7EvaluatorV1` and `EvaluationCatalogueV8EvaluatorV1` are used nowhere.

Reply:
Yes, exactly. It was all made to appear similar just to avoid raising concerns. :)